Makefile

```diff
@@ -259,6 +260,13 @@ build-stratis-base32-decode:
 # so we use two distinct targets to build the two binaries
 build-udev-utils: build-stratis-str-cmp build-stratis-base32-decode
 
+## Build the test extras
```

Comment:
Please add building with these flags to clippy Makefile target and to .PHONY list also.